samples/Test.AppWriter/Program.cs

```diff
@@ -1,16 +1,23 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Collections.Generic;
+using System.CommandLine;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.PowerPlatform.PowerApps.Persistence.Extensions;
 using Microsoft.PowerPlatform.PowerApps.Persistence.Models;
 using Microsoft.PowerPlatform.PowerApps.Persistence.MsApp;
-using Microsoft.PowerPlatform.PowerApps.Persistence.Templates;
 
 namespace Test.AppWriter;
 
 internal class Program
 {
+    //    private struct ControlsInfo
+    //    {
+    //        public int numScreens;
+    //        public string[] controls;
+    //    };
```

Comment:
Commented out code should generally be removed, as its purpose is not clear to future maintainers (often including oneself several months down the line).

If there is a reason to keep it, such as a feature temporarily disabled or as an example or reminder, it's helpful to also put an explanatory comment above it.